ZFSin/zfs/lib/libzfs/libzfs_sendrecv.c

```diff
@@ -137,7 +137,7 @@ ssread(void *buf, size_t len, FILE *stream)
 {
 	size_t outlen;
 #ifdef WIN32
-	/* Windows we have a SOCKET type here*/
+	/* Windows we have a SOCKET type here - can't change fread() call. */
 	DWORD rv;
 	if (!ReadFile(stream, buf, len, &rv, NULL))
 		return 0;
```

Comment:
Here is one of the remaining `#ifdef _WIN32`. Unix variant uses `fread()` as they are using actual `FILE *` - whereas we use `SOCKET`. Technically, it could call `read()` here, as that calls `wosix_read()` - but I think that would be more confusing. So it is more clear (I think) to call `ReadFile()` to highlight the special case.